.cirrus.yml

```diff
@@ -665,3 +665,65 @@ docker_fedora33_task:
   packages_artifacts:
     path: build/spicy-*.tar.gz
     type: application/gzip
+
+clang11_zeek_nightly_task:
+  # This task is only executed on master.
+  # Failing this task does not fail the build.
+  only_if: $CIRRUS_BRANCH == 'master'
+  allow_failures: $CIRRUS_BRANCH == 'master'
```

Comment:
Could this just be`true` given that `only_if` limits already where it runs?

Reply:
Given [the docs](https://cirrus-ci.org/guide/writing-tasks/#failure-toleration) I wasn't sure what syntax this expected. If this is just a shell expression (`true` command) or YAML (`true` literal), using `true` here should work. I adjusted the patch, let's see if it is accepted by Cirrus (it passes `cirrus validate` for me locally).